docs/user_guide/customizing.rst

```diff
@@ -65,37 +67,55 @@ For a complete list of the theme variables that you may override, see the
+    both better performance, and better script/glyph coverage than custom fonts,
+    and is recommended in most cases.
+
+The default body and header fonts can be changed as follows:
 
-- "normal" body text, on ``body``
-- page and section headers, on ``.header-style``
-- icons, on ``.fa, .far, .fas``
+- Using :ref:`custom-css`, you can specify which fonts to use for body, header
+  and monospace text. For example, the following can be added to a custom
+  css file:
 
-While altering the icon font is presently somewhat involved, the body and header fonts,
-often paired together, can be replaced (or removed altogether) by:
+  .. code-block:: none
 
-- configuring `template_path <https://www.sphinx-doc.org/en/master/theming.html#templating>`__
-  in your ``conf.py``
-- creating a custom ``layout.html`` Jinja2 template which overloads the ``fonts`` block
+      :root {
+          --pst-font-family-base: Verdana, var(--pst-font-family-base-system);
+          --pst-font-family-heading: Cambria, Georgia, Times, var(--pst-font-family-base-system);
+          --pst-font-family-monospace: Courier, var(--pst-font-family-monospace-system);
+      }
 
-.. code-block:: html+jinja
+  The ``-system`` variables are available to use as fallback to the default fonts.
 
-    {% extends "pydata_sphinx_theme/layout.html" %}
+- If the font you want to specify in the section above is not generally available
+  by default, you will additionally need to ensure the font is loaded.
+  For example, you could download and vendor the font in the ``_static`` directory
+  of your sphinx site, and then update the base template to load the font resources:
 
-    {% block fonts %}
+  - configure the `template_path <https://www.sphinx-doc.org/en/master/theming.html#templating>`__
+    in your ``conf.py``
+  - creating a custom ``layout.html`` Jinja2 template which overloads the ``fonts`` block
+    (example for loading the Lato font that is included in the ``_static/vendor`` directory):
+
+    .. code-block:: html+jinja
+
+      {% extends "pydata_sphinx_theme/layout.html" %}
+
+      {% block fonts %}
         <!-- add `style` or `link` tags with your CSS `@font-face` declarations here -->
-        <!-- ... and a `style` tag with setting `font-family` in `body` and `.header-style` -->
         <!-- ... and optionally preload the `woff2` for snappier page loads -->
-        <!-- or add a `style` tag with a font fallback chain with good cross-platform coverage -->
-        <style>
-            body {
-                font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
-            }
-            .header-style {
-                font-family: Cambria, Cochin, Georgia, Times, 'Times New Roman', serif;
-            }
-        </style>
-    {% endblock %}
+        <link rel="stylesheet" href="{{ pathto('_static/vendor/lato_latin-ext/1.44.1/index.css', 1) }}">
+
+      {% endblock %}
```

Comment:
Perhaps concretely add:

    To reduce the `Flash of Unstyled Content`, you may wish to explore various options for 
    `preloading content <https://developer.mozilla.org/en-US/docs/Web/HTML/Preloading_content>`__, 
    specifically the binary font files. This ensure the files will be loaded before waiting for the CSS to be
    parsed, and should be used with care.

and then in the code block
       
       <link rel="preload" href="{{ pathto('_static/vendor/lato_latin-ext/1.44.1/files/lato-latin-ext-100-italic.woff2', 1) }}" as="font" type="font/woff2" crossorigin>